pkg/controller/user-cluster-controller-manager/resources/reconciler.go

```diff
@@ -765,6 +778,8 @@ type reconcileData struct {
 	mlaGatewayCACert *resources.ECDSAKeyPair
 	userSSHKeys      map[string][]byte
 	cloudConfig      []byte
+	// CSICloudConfig is currently used only by vSphere, whose needs it to properly configure the external CSI driver
+	CSICloudConfig []byte
```

Comment:
Tiny nit: is there a reason why this field is exported, while others aren't? This struct is not exported anyways, but I think it would be nice to be consistent.

Reply:
You are right, gonna make it unexported.